app/api/tickets.py

```diff
@@ -247,6 +249,14 @@ def before_update_object(self, ticket, data, view_kwargs):
                 "Can't delete a ticket that has sales",
             )
 
+        if data.get('event') and ticket.sales_ends_at > event.ends_at:
```

Comment:
event does not exist here. Causing https://github.com/fossasia/open-event-frontend/issues/6218

Reply:
Fixed https://github.com/fossasia/open-event-server/pull/7561